Dockerfile

```diff
@@ -8,19 +8,10 @@ ENV PHPMYADMIN_DBPORT 3306
 # a special database for phpMyAdmin for advanced features.
 
 RUN apt-get update -q -q && \
- apt-get install phpmyadmin --yes --force-yes && \
+ apt-get install phpmyadmin php7.2-mbstring --yes && \
  sed -i 's/!empty(\$dbname)/TRUE/' /etc/phpmyadmin/config.inc.php && \
  chown -Rh :fcgi-php /var/lib/phpmyadmin && \
  chown -Rh fcgi-php /var/lib/phpmyadmin/tmp && \
- for file in /etc/php5/mods-available/*.ini; do php5enmod $(basename -s .ini "$file"); done
-
-COPY ./patches patches
-
-RUN \
- apt-get install patch --yes --force-yes && \
- for patch in patches/*; do patch --prefix=./patches/ -p0 --force "--input=$patch" || exit 1; done && \
- rm -rf patches && \
- apt-get purge patch --yes --force-yes && \
- apt-get autoremove --yes --force-yes
+ for file in /etc/php/7.2/mods-available/*.ini; do phpenmod $(basename -s .ini "$file"); done
```

Comment:
Remind me, why is this necessary?

Reply:
Removed the patch because it didn't apply. For line 15, I don't know whether it's necessary - I just changed the previous line 15 (which referred to php5) to get it to work.

Reply:
Path was integrated now.